src/main/js/models/helpers/getDisplayValue.js

```diff
@@ -106,6 +106,11 @@ const getConcatValue = (selector) => (column) => value => {
   return applyToAllLangs(lt => format(column, f.map(f.get(lt), displayValues)));
 };
 
+const index2columnId = (column) => {
+    const rows = f.get("groups", column);
+    return f.chain(rows).map(row => row.id).value();
```

Comment:
`return f.map("id", rows)` sollte es auch tun.

Generell ist es empfehlenswerter, bei Lodash/FP Daten nicht in eine Chain-Monade mit beschränkten Verkettungsmöglichkeiten zu wrappen, sondern `d = f.flow( a->b, b->c, c->d,...)(a)` oder die Abkürzung `d = doto(a, a->b, b->c, c->d,...)` zu nutzen.

Reply:
Um es ganz knapp und fp-idiomatisch zu halten könntest du auch schreiben
`const index2columId = f.flow(get("groups"), f.map("id"))`

Reply:
Dazu wäre zu sagen, dass dieses `f.flow(...)` eine Funktion zurück gibt, die ein Argument erwartet. Ausgeschrieben wäre es so:
```
const index2columId = (column) => f.flow(f.get("groups"), f.map("id"))(column);
```

Reply:
Puh, alles viel zu kompliziert gemacht. Dabei hab ichs eh schon sehr vereinfacht 😅
Ich hab nun den Index mit rein gezogen. Auch den Index in der `flow` Methode hab ich nicht geschafft.